File: src/server/Automat.py
class Automat():
    """The implementation of a exemplary automat class.
    """

    def __init__(self):
        self._current_state_id = 0

    def get_state(self):
        """Reads out the state."""
        return self._current_state_id

    def set_state(self, state_id):
        """Sets the state."""
        self._current_state_id = state_id

File: frontend/src/api/Automat.js
export default class Automat {

    // default null constructor
    constructor(aState) {
        this.state = aState;
    }

    // setting the state
    setState(aState) {
        this.state = aState;
    }

    // getting the state
    getState() {
        return this.state;
    }


}